Source/Private/common.ps1

```diff
@@ -70,15 +73,20 @@ function _callAPI {
          $params.Add('ContentType', 'application/json; charset=utf-8')
       }
 
-      if (_useWindowsAuthenticationOnPremise) {
-         $params.Add('UseDefaultCredentials', $true)
-         $params.Add('Headers', @{ })
-      }
-      elseif (_useBearerToken) {
-         $params.Add('Headers', @{Authorization = "Bearer $env:TEAM_TOKEN" })
-      }
-      else {
-         $params.Add('Headers', @{Authorization = "Basic $env:TEAM_PAT" })
+      #dont use header when requested. Then bearer must be provided with additional headers
+      $params.Add('Headers', @{ })

```

Comment:
@DarqueWarrior I rearranged it so it works better. The header is always added (also before) but now I make sure it is there and empty and then gets edited if needed.

Reply:
As long as everything still works as expected I am ok.

Reply:
It does.